job-dsls/jobs/kogito.groovy

```diff
@@ -45,7 +45,12 @@ pipeline {
            steps {
                build job: "kogito-cloud-${mainBranch}", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'mainBranch', value: mainBranch], [$class: 'StringParameterValue', name: 'ghOrgUnit', value: ghOrgUnit]] 
            }
-       }       
+       } 
+       stage('kogito-cloud-s2i-images') {
+           steps {
+               build job: "kogito-cloud-${mainBranch}", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'mainBranch', value: mainBranch], [$class: 'StringParameterValue', name: 'ghOrgUnit', value: ghOrgUnit]] 
```

Comment:
I guess you need to name the job differently, if I understand correctly. You plan to have 2 jobs there, right? kogito-cloud and kogito-cloud-s2i-images?

Reply:
correct.